test-resources/src/test/resources/html/rdfa/basic.html

```diff
@@ -14,7 +14,7 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
Seems like a strange addition. Can you explain?

Reply:
The` lang` tag is used in HTML files, and the language of XHTML files is provided using `xml:lang`. So,  since this is a HTML page, it needs to use the lang tag. Current XML parsers can use both lang or xml:lang, but since librdfa uses an old library for parsing XML it generates an error since it cannot identify the language.

Reply:
Thats fine thanks for the explanation.